runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -196,6 +237,54 @@ else if (!tagSearch)
 		}
 	}
 
+	@Subscribe
+	public void onMenuOpened(MenuOpened event)
```

Comment:
I would stick with modifying the string on the script stack probably instead of this.